src/transport/FabricTable.h

```diff
@@ -41,7 +41,7 @@
 namespace chip {
 
 static constexpr FabricIndex kMinValidFabricIndex     = 1;
-static constexpr FabricIndex kMaxValidFabricIndex     = std::min(UINT8_MAX, CHIP_CONFIG_MAX_DEVICE_ADMINS);
+static constexpr FabricIndex kMaxValidFabricIndex     = std::min<FabricIndex>(UINT8_MAX, CHIP_CONFIG_MAX_DEVICE_ADMINS);
```

Comment:
```suggestion
static constexpr FabricIndex kMaxValidFabricIndex     = std::min<FabricIndex>(UINT8_MAX-1, CHIP_CONFIG_MAX_DEVICE_ADMINS);
```

The spec only allows 1..254 now due to a uint8 type range issue.

Reply:
ack. thanks for the change.